modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,29 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.26]]
+== Neo4j 5.26
+
+=== Deprecated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CALL db.schema.nodeTypeProperties() YIELD propertyTypes RETURN propertyTypes;
+CALL db.schema.relTypeProperties() YIELD propertyTypes RETURN propertyTypes;
+----
+a|
+The column `propertyTypes` currently produces a list of strings representing the possible Java types the given property has, in an upcoming major release it will instead represent the possible Cypher Types the given property has.
+For all available Cypher types, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```

Comment:
```suggestion
The column `propertyTypes` currently produces a list of strings representing representing the potential Java types for a given property.
In an upcoming major release of Neo4j, this will be updated to represent the possible Cypher types for that property instead.
For all available Cypher types, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```